plugins/test/runner.proto

```diff
@@ -147,7 +147,13 @@ message Env {
   // Log file to receive plugin output. Can be /dev/std{out,err}.
   string log_path = 5;
   // Fixed timestamp in Unix seconds. If unset it will be the Unix epoch.
-  uint64 time_secs = 6;
+  optional uint64 time_secs = 6;
+  // Number of background streams to run in benchmarks. If unset, no background
+  // streams will be run.
+  optional uint64 num_background_streams = 7;
+  // If there are background streams, this controls the number of background
+  // streams that are advanced per iteration of the benchmark loop.
+  optional uint64 background_stream_advance_rate = 8;
```

Comment:
The concept of 'advance rate' is a bit unintuitive to me. I think it means 'for every 1 CPU-measured cycle, we also run N CPU-unmeasured cycle-phases'. What's tricky is that this doesn't control the number of cycles in total, nor the number of phases executed in the main cycle.

I don't have any great suggestions here other than expanding the comment and/or calling it a 'ratio'.

Reply:
I started to rename it to "ratio", but then that field a little weird to me since the value is an int, and ratio seems like it should allow floats / fractions. So I left the name as-is but expanded the comment.